Sources/Identity/CustomerInfoManager.swift

```diff
@@ -334,11 +380,13 @@ extension CustomerInfoManager {
 
     func customerInfo(
         appUserID: String,
+        trackDiagnostics: Bool,
```

Comment:
I wonder if we should default this to `false`, since seems to be the main use of this and just set it to `true` where needed... NABD

Reply:
Yes, I went back and forth with this one and I couldn't really make up my mind. So I agree that it's cleaner if it's going to be `false` most of the times anyway